CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Introduce [CSSComb](https://github.com/csscomb/csscomb.js). For more details see: https://github.com/fs/rails-base/pull/284
 - Add `Brewfile` to track application dependencies.
+- Fixed rails-best-practices configuration file using
```

Comment:
@vladimirbazhanov could you please move this entry to the top and change it a little bit:
> Fix `rails_best_practices` invocation from `bin/quality`, `config/rails_best_practices.yml` was ignored before.